content/en/guide/v10/differences-to-react.md

```diff
@@ -24,10 +24,10 @@ For us it doesn't make sense as the browser's event system supports all features
 
 We've come across the following differences between React's synthetic event system and native browser events:
 
-- Events don't bubble through `<Portal>` components
+- Events don't bubble through `<Portal>` components.
 - The clear "x" button for `<input type="search">` does not fire an `input` event in IE11 - use `onSearch` instead.
-- Use `onInput` instead `onChange` for `<input>` elements (**only if `preact/compat` is not used**)
-- Use `onDblClick` instead of `onDoubleClick` (**only if `preact/compat` is not used**)
+- Use `onInput` instead of `onChange` for `<input>` elements (**only if `preact/compat` is not used**).
+- Use `onDblClick` instead of `onDoubleClick` (**only if `preact/compat` is not used**).
```

Comment:
I'd go the other way with the periods, personally. These aren't really sentences, but fragments. 

Removing the period on the odd item out (`The clear "x"...`) I think makes more sense than adding to every other item.

Reply:
Sounds good to me. I'll remove the periods from the bullets. I made the commit and pushed it up to my branch, but it looks like GitHub is behaving strangely and not updating this PR to show that new commit. I'm going to try closing this PR and open a new one.